lib/beaker/hypervisor/docker.rb

```diff
@@ -416,7 +416,6 @@ def install_ssh_components(container, host)
         container.exec(%w(ssh-keygen -t dsa -f /etc/ssh/ssh_host_dsa_key))
         container.exec(%w(sed -ri 's/^#?UsePAM .*/UsePAM no/' /etc/ssh/sshd_config))
       when /archlinux/
-        container.exec(%w(pacman --noconfirm -Sy archlinux-keyring))
```

Comment:
Is this line no longer needed or was this unintended?

Reply:
I cleaned up the parameters and now use the long options. it basically does the same the the RUN commands in the docker section.

Reply:
The whole idea of this method is that it can run when there is a Dockerfile. It should not get executed when there is none:
https://github.com/voxpupuli/beaker-docker/blob/b49e259f7da107f670115f37ea5f20e6629377f2/lib/beaker/hypervisor/docker.rb#L54-L68
That method is poorly named, but it gets called here:
https://github.com/voxpupuli/beaker-docker/blob/b49e259f7da107f670115f37ea5f20e6629377f2/lib/beaker/hypervisor/docker.rb#L352-L357

So if there is a `dockerfile` for the host (or `use_image_entry_point`), it's assumed the `RUN` commands weren't executed and still need to happen. That's why the duplication is there.

Reply:
okay, so this is a 100% duplication or meant for commands that need to be executed after the container is started properly? It should be fine to merge this now and afterwards, if someone has time, move most of the stuff into run commands for every OS?

Reply:
okay. at the moment the RUN commands for Arch Linux provide a working image, same for the explicit commands. I think we can merge that now and have a discussion if/how we want to support custom image or if we want to deduplicate it?

Reply:
I'm leaning to not touching this line at all (so putting it back). Then I'd also take a critical look at the commit messages and what this PR is really achieving. Then I'd be fine with merging it.